Automattic/newspack-custom-content-migrator
# Fix/moving content from a category to another

## How to test
- Create two categories "Source Category", and "Destination Category".
- Create Three posts, one for each category, and the third for both categories.
- Run the command to move posts from the "Source Category" to the "Destination Category": `wp newspack-content-migrator move-content-from-one-category-to-another --source-term-id=123 --destination-term-id=456`
- Notice the SQL error and that the posts were not moved.
- Switch to this branch and re-run the command.
- Notice The Source Category with no posts, and all the posts were moved to the Destination Category.

Play a bit more with it 😅 

---
